code/src/OSFramework/Feature/IConditionalFormat.ts

```diff
@@ -0,0 +1,9 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+namespace OSFramework.Feature {
+    export interface IConditionalFormat {
+        addRule(
```

Comment:
Please add some comments here in order to make it easier to understand the behavior.

Reply:
done